examples/lines_bars_and_markers/markevery_demo.py

```diff
@@ -46,47 +46,42 @@
 ###############################################################################
 # Plot each markevery case for linear x and y scales
 
-fig1 = plt.figure(num=1, figsize=figsize)
-ax = []
+fig1, axs = plt.subplots(rows, cols, figsize=figsize, constrained_layout=True)
+axs = axs.flat
+axs[-1].remove()  # not used...
```

Comment:
That depends on the length of cases, doesn't it?

Reply:
yeah, but...

Reply:
something like `for ax in axs[:len(cases)]: ax.remove()` after converting to axs.flat looks better to me.